src/Queue.php

```diff
@@ -56,9 +56,15 @@ public function push($job, $data = '', $queue = null)
      */
     public function pushRaw($payload, $queue = null, array $options = [])
     {
+        $message = $this->psrContext->createMessage($payload);
+
+        if ($message instanceof \Interop\Amqp\Impl\AmqpMessage) {
```

Comment:
could you import namespace.

Reply:
For interface `\Interop\Amqp\AmqpMessage` or implementation `Interop\Amqp\AmqpMessage`?
We can't use both or I could use an alias. Which you prefer?

Reply:
use Interop\Amqp\AmqpMessage;

if ($message instanceof AmqpMessage) {